### PR TITLE
fix(redirects): use cached redirects document

### PR DESCRIPTION
**Pull request by tripodsan**
use cached redirects.json. flush needs currently to be done manually.
(which is not a problem, according to @davidnuescheler )

### CODE REVIEW

**Review by trieloff — APPROVED** [review]

File: helix-redirects.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 redirects:
-  - https://adobe.sharepoint.com/:x:/s/TheBlog/EaikC7JAUNpAoZy604FUP7YB2oTdxZVtrWNihbJ6k-BCnA?email=&e=Nd2OF4
+  - https://master--theblog--adobe.hlx.page/redirects.json
```
Comment (trieloff) [inline helix-redirects.yaml:2]: Are you sure that this works?

Comment (trieloff) [reply helix-redirects.yaml:2]: Nevermind, I forgot about https://github.com/adobe/helix-shared/pull/438

Comment (tripodsan) [reply helix-redirects.yaml:2]: it should :-) but I can test it....

Comment (tripodsan) [reply helix-redirects.yaml:2]: hmmm...it doesn't work and helix-redirect is not very verbose :-(

Comment (tripodsan) [reply helix-redirects.yaml:2]: https://github.com/adobe/helix-shared/issues/445

**Review by rofe — APPROVED** [review]

File: helix-redirects.yaml (hunk 1 above)
Comment (tripodsan) [reply helix-redirects.yaml:2]: ok works:

```
$ curl -sD - 'https://lqmig3v5eb.execute-api.us-east-1.amazonaws.com/helix-services/redirect/v1?owner=tripodsan&repo=helix-pages-test&ref=master&path=/tag/coronavirus/'
HTTP/2 301
date: Wed, 03 Mar 2021 11:39:15 GMT
content-type: text/plain; charset=utf-8
content-length: 17
location: https://blog.adobe.com/en/topics/covid-19.html
cache-control: max-age=30000000
apigw-requestid: bm9G7h-jIAMEVlw=

moved permanently
```

**Pull request merged**